docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+networks:
+  xpd-network:
+    external: false
+
+services:
+  xpd-gateway:
+    image: ghcr.io/randomairborne/xpd-gateway:latest
+    restart: unless-stopped
+    networks:
+      - xpd-network
+    env_file:
+      - .env
+    environment:
+      - REDIS_URL=redis://redis:6379
```

Comment:
Do we maybe want to set the postgres URL automatically based on the POSTGRES_PASSWORD? We have all the other information.